res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe

```diff
@@ -31,6 +31,11 @@ operand {
     dim: 0
     dim: 512
   }
+  shape_signature {
+    dim: 1
+    dim: -1
+    dim: 512
+  }
```

Comment:
We may have to revise comment

> This was generated by tflchef-reverse from res/TensorFlowTests/NET_0043

Reply:
Thanks for pointing out. I've updated the comment.